### PR TITLE
Replace ERB templates with HAML for webui/search controller views

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

File: src/api/app/views/webui/search/project.html.erb [hunk 1]
```diff
@@ -1,58 +0,0 @@
-<% require 'active_support/inflector' %>
```
Comment (DavidKang) [inline src/api/app/views/webui/search/project.html.erb]: Why did you remove this file? Did you convert it to HAML? 🤔

Comment (mdeniz) [inline src/api/app/views/webui/search/project.html.erb]: is unused as far as I saw... if you want to check that 😄

Comment (DavidKang) [inline src/api/app/views/webui/search/project.html.erb]: cool! 👍
